static/src/js/actions/__tests__/viewAllFacets.test.js

```diff
@@ -0,0 +1,436 @@
+    })
+    expect(storeActions[1]).toEqual({
+      type: UPDATE_COLLECTION_QUERY,
+      payload: {
+        pageNum: 1
+      }
+    })
+    expect(storeActions[2]).toEqual({
+      type: UPDATE_SELECTED_CMR_FACET,
+      payload: {
+        data_center_h: undefined,
+        instrument_h: ['1 Test facet', 'Test facet 2'],
+        platform_h: undefined,
+        processing_level_id_h: undefined,
+        project_h: undefined,
+        science_keywords_h: undefined
+      }
+    })
+  })
+})
+
+describe('copyCMRFacets', () => {
+  test('should create an action to update the View All Facets state', () => {
+    const store = mockStore({
+      ui: {
+        facetsModal: {
+          isOpen: true
+        }
+      },
+      facetsParams: {
+        cmr: { instrument_h: ['1 Test facet', 'Test facet 2'] },
+        viewAll: {}
+      },
+      query: {
+        collection: {}
+      },
+      entities: {
+        viewAllFacets: {}
+      }
+    })
+
+    store.dispatch(copyCMRFacets())
+    const storeActions = store.getActions()
+    expect(storeActions[0]).toEqual({
+      type: COPY_CMR_FACETS_TO_VIEW_ALL,
+      payload: { instrument_h: ['1 Test facet', 'Test facet 2'] }
+    })
+  })
+})
+
+describe('getCollections', () => {
+  beforeEach(() => {
+    moxios.install()
+
+    jest.clearAllMocks()
+  })
+
+  afterEach(() => {
+    moxios.uninstall()
+  })
+
+  test('calls the API to get the View All Facets', async () => {
+    moxios.stubRequest(/collections.*/, {
+      status: 200,
+      response: {
+        feed: {
+          updated: '2019-03-27T20:21:14.705Z',
+          id: 'https://cmr.sit.earthdata.nasa.gov:443/search/collections.json?has_granules_or_cwic=true&include_facets=v2&include_granule_counts=true&include_has_granules=true&include_tags=edsc.%2A%2Corg.ceos.wgiss.cwic.granules.prod&keyword=&options%5Btemporal%5D%5Blimit_to_granules%5D=true&page_num=1&page_size=20&sort_key=has_granules_or_cwic&facets_size[instrument]=10000',
+          title: 'ECHO dataset metadata',
+          entry: [{
+            mockCollectionData: 'goes here'
+          }],
+          facets: {
+            children: {
+              instrument: {
+                applied: true,
+                has_children: true,
+                title: 'Instrument',
+                type: 'group',
+                children: [
+                  {
+                    title: '1 Test facet',
+                    type: 'filter',
+                    applied: false,
+                    has_children: false,
+                    links : {
+                      apply: 'https://cmr.sit.earthdata.nasa.gov:443/search/collections.json?has_granules_or_cwic=true&include_facets=v2&include_granule_counts=true&include_has_granules=true&include_tags=edsc.%2A%2Corg.ceos.wgiss.cwic.granules.prod&keyword=&options%5Btemporal%5D%5Blimit_to_granules%5D=true&page_num=1&page_size=20&sort_key=has_granules_or_cwic&facets_size[instrument]=10000&instrument_h%5B%5D=1+Test+facet'
```

Comment:
@abbottry This did not work. I think we may need to poke around CMR docs, but dont want to hold this ticket up any longer. Ill do some research and see if this can be improved down the line.